src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -45,9 +45,10 @@
           inputFileLabel="Continuation authorization"
           :maxSize="MAX_FILE_SIZE"
           :pdfPageSize="PdfPageSize.LETTER_SIZE"
+          :hint="authorization.files[index]?.file ? 'File uploaded.' : undefined"
```

Comment:
Shows hint only if file is defined. Screenshot:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/7312fe3e-b30c-4728-a173-6d16862c6f58)